src/DebuggingFeature.ts

```diff
@@ -0,0 +1,77 @@
+import { BaseLanguageClient, StaticFeature, Neovim } from "coc.nvim";
+import {
+  ExecuteCommandParams,
+  ExecuteCommandRequest,
+  InitializeParams,
+  ServerCapabilities,
+} from "vscode-languageserver-protocol";
+import { commands, workspace } from "coc.nvim";
+
+interface DebugAdapterStartResponse {
+  name: string;
+  uri: string;
+}
+
+export class DebuggingFeature implements StaticFeature {
+  private withVimspector = false;
+
+  constructor(private _nvim: Neovim, private _client: BaseLanguageClient) {}
+
+  async preInit(): Promise<void> {
+    this.withVimspector = (await this._nvim.getVar("loaded_vimpector")) === 1;
+  }
+
+  fillInitializeParams(params: InitializeParams): void {
+    if (params.capabilities.experimental == null) {
+      params.capabilities.experimental = {};
+    }
+    (params.capabilities.experimental as any).debuggingProvider =
+      workspace.isNvim && this.withVimspector;
+  }
```

Comment:
Recently we switched over from setting a lot of the configuration via `ClientExperimentalCapabilities` to `InitializationOptions`. That's now the preferred way to set these in Metals. So are we able to refactor this to instead simply pass in `debuggingProvider: true` rather than going the experimental route?

https://github.com/scalameta/coc-metals/blob/82c4ace4e37a06321c886008fdc37f102cb90cc5/src/index.ts#L183-L192

Reply:
It seems I did. Please check.